developers/pymavlink/request_message_interval.py

```diff
@@ -0,0 +1,35 @@
+"""
+Example of how to request a message in a desired interval
+"""
+
+# Disable "Bare exception" warning
+# pylint: disable=W0702
+
+import time
+# Import mavutil
+from pymavlink import mavutil
+
+# Create the connection
+master = mavutil.mavlink_connection('udpin:0.0.0.0:14550')
+# Wait a heartbeat before sending commands
+master.wait_heartbeat()
+
+# Configure AHRS2 message to be sent at 1Hz
```

Comment:
we should explain that for 1Hz, the period is 1e6 microsseconds. I have mixed feelings about using the `1e6` notation for users, too...
It should be fine if we add comments.

Reply:
moved to function